jobbergate-api/alembic/versions/20240226_175811--35ccf022687d_replace_slurmrestd_by_sbatch.py

```diff
@@ -0,0 +1,41 @@
+"""Replace slurmrestd by sbatch
+
+Revision ID: 35ccf022687d
+Revises: ec2d2948fb41
+Create Date: 2024-02-26 17:58:11.147803
+
+"""
+
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "35ccf022687d"
+down_revision = "ec2d2948fb41"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("job_submissions", sa.Column("sbatch_arguments", sa.String(), nullable=True))
+    op.drop_column("job_submissions", "template_vars")
```

Comment:
I think you should apply some logic here to move the template_vars to the sbatch params format before just dropping the column

Reply:
It is possible, but not trivial. This PR deletes the parser that goes from SBATCH to execution parameters with +300 lines of code, we would need to code it in the opposite direction and keep it in this migration file. In addition, not all of them have a one-on-one relation, so we would need to cover some edge cases.

I also consider that dropping it won't affect any completed job.

Reply:
wait, why is template_vars being dropped?

Reply:
Please, see this note: https://github.com/omnivector-solutions/jobbergate/pull/497#discussion_r1508172295

Reply:
> wait, why is template_vars being dropped?

Please, see this note: https://github.com/omnivector-solutions/jobbergate/pull/497#discussion_r1508172295

Reply:
I see. The column was incorrectly named in the first place. Now, we are dropping the column so we need to use the erronious name to do so. This will actually kill two birds with one stone.

Reply:
We agreed we were okay with just dropping the previous column.